dcc-portal-ui/app/scripts/analysis/js/analysis.js

```diff
@@ -166,7 +166,7 @@
           $scope.analysisResult = null;
           $timeout(function() {
             $scope.analysisResult = data;
-          }, 150);
+          }, 250);
```

Comment:
Should we add a comment to explain magic number timeouts?
Is it at all possible to avoid the timeout?